cider/exec/module/CiderTypes.cpp

```diff
@@ -24,10 +24,13 @@
 #include "cider/CiderTypes.h"
 #include "util/DateTimeParser.h"
 
-int64_t CiderDateType::toInt64(const std::string& string_val) {
+int64_t DateAndTimeType::toInt64(const std::string& string_val, Type type) {
+  DateTimeParser::FormatType fmtType = (type == Type::Time)
```

Comment:
what is the difference between time and timestamp? why doesn't there contain timestamp?

Reply:
time since the beginning of any day, Range of [0..86,399,999,999], while timestamp within [1000-01-01 00:00:00.000000..9999-12-31 23:59:59.999999], with microsecond precision. And as the  temporal scalar function support, the two types are also different.

Reply:
So for `DateTimeParser::FormatType::TimeStamp`, don't need any code change here?

Reply:
timestamp string format fixed and add test.